bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx126beta-en-GB.html

```diff
@@ -0,0 +1,51 @@
+{#
+  This Source Code Form is subject to the terms of the Mozilla Public
+  License, v. 2.0. If a copy of the MPL was not distributed with this
+  file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ #}
+
+{% extends "firefox/whatsnew/base.html" %}
+
+{% block page_title %}What’s new with Firefox{% endblock %}
+
+{#- This will appear as <meta property="og:description"> which can be used for social share -#}
+{% block page_og_desc %}{{ ftl('whatsnew-page-description') }}{% endblock %}
+
+{% block body_id %}firefox-whatsnew{% endblock %}
+
+{% block page_css %}
+  {{ css_bundle('firefox_whatsnew_126beta') }}
+{% endblock %}
+
+{% block site_header %}{% endblock %}
+
+{% block wnp_content %}
+  <section class="wnp-content-main">
+    <div class="mzp-l-content mzp-t-content-sm">
+      <aside class="wnp-swap">
+        <div class="wnp-swap-blurb-here">
+          <span class="wnp-swap-text-here" lang="en">hello 🫖</span>
+          {% include 'firefox/whatsnew/includes/fx126beta/arrow-blue.svg' %}
+        </div>
+        <div class="wnp-swap-blurb-there">
+          <span class="wnp-swap-text-there" lang="fr">bonjour 🥖</span>
+          {% include 'firefox/whatsnew/includes/fx126beta/arrow-white.svg' %}
```

Comment:
I'm seeing inconsistent placement of the arrow SVG files on each page?

![Screenshot 2024-04-11 at 4 01 02 PM](https://github.com/mozilla/bedrock/assets/42309026/c875d00e-bfc0-4895-8419-6d515ecad647)

Reply:
weird... I can't reproduce this. Wonder what's happening...